### ISSUE TITLE
java.lang.IllegalStateException: Fragment

### ISSUE THREAD

**Issue by auxDK** [Netvent/storyly-mobile#148]
Issue tracker is **ONLY** used for reporting Android SDK related bugs.

## Environment Details
Storyly SDK Version:1.18.3
Android SDK Version:Any
Device Model:Any
Device OS Version:Any

## Expected Behavior

No crash 

## Current Behavior

Crash app 

```
Fatal Exception: java.lang.IllegalStateException: Fragment already added: c{b016434} (7239b058-0b88-4e65-a3d7-344c6baaeafc tag=StorylyDialogFragment)
       at androidx.fragment.app.FragmentStore.addFragment(FragmentStore.java:91)
       at androidx.fragment.app.FragmentManager.addFragment(FragmentManager.java:1713)
       at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:415)
       at androidx.fragment.app.FragmentManager.executeOps(FragmentManager.java:2404)
       at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2162)
       at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:2100)
       at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:2002)
       at androidx.fragment.app.FragmentManager$5.run(FragmentManager.java:524)
       at android.os.Handler.handleCallback(Handler.java:873)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:201)
       at android.app.ActivityThread.main(ActivityThread.java:6861)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:547)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:873)
```

**Comment by levent-oral** [comment]
Hello @auxDK, 

Can you give us information about steps to reproduce? 

Thanks,

**Comment by auxDK** [comment]
Hi @levent-oral,
I do not have steps, I`m getting this log from firebase crashlytics

![Screenshot from 2022-01-11 16-18-48](https://user-images.githubusercontent.com/2931103/148959716-185ec59f-a6f7-48e2-b056-f7182b21f368.png)

**Comment by levent-oral** [comment]
Hello @auxDK ,

We've released [Storyly Android SDK 1.19.2](https://github.com/Netvent/storyly-mobile/blob/master/android/RELEASENOTES.md#1192-1622022) which contains major improvements for this type of issues. Please check with it and let us know if the issue persists.

Thanks,

**Comment by auxDK** [comment]
Hi @levent-oral  
We will release the new version in 2 weeks if the crash is not reproduced I'm closing this ticket
